src/components/App/App.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {render, screen,fireEvent,within } from "@testing-library/react"
+import '@testing-library/jest-dom/extend-expect'
+import VideoContext from "../../providers/VideoContext";
+import {BrowserRouter as Route} from "react-router-dom";
+import App from "./App"
+describe("<App/>",()=> {
+   
+    test("Testing app rendering",()=> {
```

Comment:
This test only verifies if the ```App``` is created, I think you should add more tests to verify different behaviors depending the states.